docs/user-guide/src/baremetal/guide.md

```diff
@@ -0,0 +1,227 @@
+  sudo docker tag registry.nordix.org/quay-io-proxy/metal3-io/ironic:latest 192.168.111.1:5000/localimages/ironic:latest
+  sudo docker push 192.168.111.1:5000/localimages/ironic:latest
+
+  # Ironic python agent
+  sudo docker pull registry.nordix.org/quay-io-proxy/metal3-io/ironic-ipa-downloader
+  sudo docker tag registry.nordix.org/quay-io-proxy/metal3-io/ironic-ipa-downloader 192.168.111.1:5000/localimages/ironic-ipa-downloader
+  sudo docker push 192.168.111.1:5000/localimages/ironic-ipa-downloader
+
+  # MariaDB
+  sudo docker pull registry.nordix.org/quay-io-proxy/metal3-io/mariadb:latest
+  sudo docker tag registry.nordix.org/quay-io-proxy/metal3-io/mariadb:latest 192.168.111.1:5000/localimages/mariadb:latest
+  sudo docker push 192.168.111.1:5000/localimages/mariadb:latest
+
+  # Keepalived
+  sudo docker pull registry.nordix.org/quay-io-proxy/metal3-io/keepalived:latest
+  sudo docker tag registry.nordix.org/quay-io-proxy/metal3-io/keepalived:latest 192.168.111.1:5000/localimages/keepalived:latest
+  sudo docker push 192.168.111.1:5000/localimages/keepalived:latest
+
+  # Metal3 IP address manager
+  sudo docker pull registry.nordix.org/quay-io-proxy/metal3-io/ip-address-manager:main
+  sudo docker tag registry.nordix.org/quay-io-proxy/metal3-io/ip-address-manager:main 192.168.111.1:5000/localimages/ip-address-manager:main
+  sudo docker push 192.168.111.1:5000/localimages/ip-address-manager:main
+
+  # Cluster API proviser metal3
+  sudo docker pull registry.nordix.org/quay-io-proxy/metal3-io/cluster-api-provider-metal3:main
+  sudo docker tag registry.nordix.org/quay-io-proxy/metal3-io/cluster-api-provider-metal3:main 192.168.111.1:5000/localimages/cluster-api-provider-metal3:main
+  sudo docker push 192.168.111.1:5000/localimages/cluster-api-provider-metal3:main
+  ```
+
+* Start httpd container
+
+  ```bash
+  sudo docker run -d --net host --privileged --name httpd-infra -v /opt/metal3-dev-env/ironic:/shared --entrypoint /bin/runhttpd --env
+  ```
+
+## Launch management cluster using minikube
+
+* Creates minikube management cluster.
+
+  ```bash
+  minikube start
+
+  # Configuring ironicendpoint with minikube
+  minikube ssh sudo brctl addbr ironicendpoint
+  minikube ssh sudo ip link set ironicendpoint up
+  minikube ssh sudo brctl addif ironicendpoint eth2
+  minikube ssh sudo ip addr add 172.22.0.9/24 dev ironicendpoint
+  ```
+
+* Launch capm3.
+
+  ```bash
+  kubectl create namespace metal3
+  clusterctl init --core cluster-api:v1.4.2 --bootstrap kubeadm:v1.4.2 --control-plane kubeadm:v1.4.2 --infrastructure=metal3:v1.4.0 -v5
+  ```
+
+* Launch baremetal operator.
+
+  ```bash
+  # Clone BMO repo
+  git clone https://github.com/metal3-io/baremetal-operator.git
+  # Run deploy.sh
+  ./baremetal-operator/tools/deploy.sh -b -k -t
+  ```
+
+* Launch ironic.
+
+  ```bash
+  # Run deploy.sh
+  ./baremetal-operator/tools/deploy.sh -i -k -t
+  ```
+
+## Create secrets and crds for BareMetalHosts
```

Comment:
The CRDs were already installed with BMO. I think you meant the CRs, but they are BMHs so lets just call them that :slightly_smiling_face: 
```suggestion
## Create Secrets and BareMetalHosts
```